### ISSUE TITLE
images en alt tag

### ISSUE THREAD

**Issue by KoopReynders** [fdnd/visitatie#8]
Figures and images in PDF documents should have non blank ALT text, except for decorative images which should be marked as artifacts. 

Each image should have an ALT attribute describing the picture, which screen readers can read aloud.
